package/libs/openssl/Config.in

```diff
@@ -1,32 +1,5 @@
 if PACKAGE_libopenssl
 
-choice
```

Comment:
Instead of introducing a revert commit, could you please drop that commit when rebasing?